examples/Connect firefly_client.ipynb

```diff
@@ -0,0 +1,269 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This notebook shows how to connect firefly_client to image and table widgets."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from firefly_widgets import connect\n",
+    "connect('http://localhost:8080/firefly')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import ipywidgets\n",
+    "from ipywidgets import Layout\n",
+    "ipywidgets.__version__"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Image widget example"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from firefly_widgets import ImageViewer"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Start a widget with the defaults, which fetch an image"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "iv = ImageViewer(GridOn=False, layout=Layout(width='400px', height='400px', plot_id='imwidget-1'))"
```

Comment:
the brace in the wrong place, should be 
ImageViewer(GridOn=False, layout=Layout(width='400px', height='400px'), plot_id='imwidget-1')"